### ISSUE TITLE
Some minor issues regarding the reproduction of CFL-Net

### ISSUE THREAD

**Issue by GWZ313** [KishoreBD/ASPP-for-Pamaap2-dataset#1]
Hello author, when I reproduced your paper CFL-Net, I divided the IMD2020 dataset and trained with the training set for 200 epochs. The final Softmax Loss: 0.061503884600317894, Contrastive Loss: 0.7723994139044439. However, when I verified with the Test set, the AUC score was approximately 9% lower than the AUC score mentioned in the paper. Could you please tell me which step went wrong? I hope you can reply.
